index.js

```diff
@@ -144,8 +148,16 @@ async.forEach(inputFiles, compile, onError);
 function fsWatcher(entryPoints) {
   var watchedFiles = entryPoints;
   var index = {}; // source files by entry point
+  var opts = {};
+
+  if (argv.poll || typeof argv.poll === 'number') {
+    opts.usePolling = true;
+    opts.interval = argv.poll !== true
+      ? argv.poll
+      : undefined;
```

Comment:
interval should be Number in the [chokidar/lib/nodefs-handler.js#150](https://github.com/paulmillr/chokidar/blob/69a549315cd862f54942c042edfd42edfabadb55/lib/nodefs-handler.js#L150). How about to update this part to the following?

```js
if (typeof argv.poll === 'number') {
  opts.usePolling = true;
  opts.interval = argv.poll;
}
```